node/src/components/consensus/era_supervisor.rs

```diff
@@ -765,6 +765,7 @@ where
                     rewards: tbd.rewards,
                     equivocators: era.accusations(),
                     inactive_validators: tbd.inactive_validators,
+                    next_era_validator_weights: BTreeMap::default(),
```

Comment:
I'd prefer using a different type in `FinalizedBlock` instead of just putting an empty map in there.

I'd actually keep `EraEnd` without `next_era_validator_weights`, and call the new structure _with_ that field `Switch` or `SwitchBlockData` or something maybe?

Reply:
Created a new struct called `SwitchBlockData` and inserted into `BlockHeader`